spark_ec2.py

```diff
@@ -162,11 +183,74 @@ def setup_external_libs(libs):
 from boto.ec2.blockdevicemapping import BlockDeviceMapping, BlockDeviceType, EBSBlockDeviceType
 from boto import ec2
 
+import yaml
 
 class UsageError(Exception):
     pass
 
 
+def process_conf_file(file_path, parser):
```

Comment:
I know that the spark_ec2.py doesn't have a lot of unit tests, but it looks to me that this is a function that can be unit tested relatively easily ? Would be great if we can add a `tests/test_yaml_config.py` to exercise various code paths